metabase/metabase
# add sunburst to pie chart

Closes https://github.com/metabase/metabase/issues/47209
Closes https://github.com/metabase/metabase/issues/8563

### Description

Adds a sunburst feature to the pie chart, using up to three dimension columns.

### How to verify

1. Create a question with three dimensions, one metric, see example below

<img width="1210" alt="Screenshot 2024-09-26 at 2 48 39 PM" src="https://github.com/user-attachments/assets/d54740f8-9fbb-4747-be23-9d4ac60a0d38">

2. Visualize as pie chart, should have automatically select dimension and metric columns

### Demo

<img width="579" alt="Screenshot 2024-09-26 at 2 49 14 PM" src="https://github.com/user-attachments/assets/cf211e84-a2c4-4995-9259-421dcfd07279">

### Checklist
